packages/node_modules/@webex/plugin-meetings/src/member/index.js

```diff
@@ -215,7 +215,6 @@ export default class Member {
       this.email = MemberUtil.extractEmail(participant);
       this.isAudioMuted = MemberUtil.isAudioMuted(participant);
       this.isVideoMuted = MemberUtil.isVideoMuted(participant);
-      this.isRecording = MemberUtil.isRecording(participant);
```

Comment:
Would it be easier to move the logic from `processIsRecording` to this function?

Reply:
No -- in this case that breaks the current pattern set up